hw/top_earlgrey/lint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -68,6 +68,11 @@
                   import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
                   rel_path: "hw/ip/i2c/lint/{tool}"
              },
+             {    name: pattgen
+                  fusesoc_core: lowrisc:ip:pattgen
+                  import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
```

Comment:
This line should be
```suggestion
                  import_cfgs: ["{proj_root}/hw/lint/tools/dvsim/common_lint_cfg.hjson"]
```

Reply:
Oh my goodness,  what a silly mistake!   Thanks for fixing that @imphil.